users/views.py

```diff
@@ -30,6 +30,13 @@ class UserProfileView(LoginRequiredMixin, DetailView):
     def get_object(self: "UserProfileView") -> User:
         return self.request.user
 
+    def get_context_data(self: "UserProfileView", **kwargs: dict[str, Any]) -> dict[str, Any]:
```

Comment:
Do we need that? It can be retrieved in template no? Using relations.

Reply:
added "receiver" to model, so now it's not needed